web_widget_numeric_step/README.rst

```diff
@@ -0,0 +1,124 @@
+
+Demo available at `Settings > Users & Companies > Users > *Select One* > See 'Credit Limit' field`
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+In your xml view, add ``widget="numeric_step"``
+This will add the 2 buttons "+" and "-" just next to the input field in edit mode.
+Iteration step by default is 1.
+
+.. figure:: https://raw.githubusercontent.com/OCA/web/13.0/web_widget_numeric_step/static/description/add_two_buttons.png
+
+
+**Optional**
+
+Add an option to choose the step iteration and limits (min and max values).
+
+Example for an 0.25 step, min to -1 and max to 10 :
+
+.. code:: xml
+
+  <field name="sale_delay" widget="numeric_step" options="{'step': 0.25, 'min': -1, 'max': 10}" /> days
+
+**Available Options**
+
+- step > Amount to increase/decrease (default: 1.0)
+- min > Min. value allowed (default: no limit)
+- max > Max. value allowed (default: no limit)
+- auto_select > Select the content when the element get focus (default: False)
+- placeholder > Define the placeholder text (default: None)
+
+**Examples**
+
+Iteration with 0.25 step, min to -1 and max to 10.
+
+Start to increment with button, continue incrementing with scrolling mouse.
+
+.. figure:: https://raw.githubusercontent.com/OCA/web/13.0/web_widget_numeric_step/static/description/step0,25andlimits.gif
+
+Iteration with 10 step, max limit 15, placeholder with onchange
+
+.. figure:: https://raw.githubusercontent.com/OCA/web/13.0/web_widget_numeric_step/static/description/step10_limit15_placeholder117_with_onchange.gif
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/web/issues/new?body=module:%20web_widget_numeric_step%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* GRAP
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `GRAP <http://www.grap.coop>`_:
+
+  * Quentin DUPONT <quentin.dupont@grap.coop>
+
+* `Tecnativa <https://www.tecnativa.com/>`_:
+
+  * Alexandre Díaz
+  * Carlos Roca
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/web <https://github.com/OCA/web/tree/13.0/web_widget_numeric_step>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```

Comment:
Please. You must re-genarate this file

Reply:
hello, @sergio-teruel can you please re-check?